05_customizing-extending-masa/06_events/02_lifecycles.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: default
+title: Lifecycles
+nav_order: 2
+grand_parent: Customizing/Extending Masa
+parent: Events
+last_modified_date: 11-01-2021 10:00
+permalink: /customizing-extending-masa/events/lifecycles/
+---
+
+# Lifecycle
+{: .no_toc }
```

Comment:
A short description of the lifecycle and what it does and how it can be used would be nice.